content/install/integrations/gitea.md

```diff
@@ -18,6 +18,11 @@ version: '2'
 services:
   drone-server:
     image: drone/drone:{{% version %}}
+    ports:
+      - 80:8000
```

Comment:
I don't think this is necessary in the doc since the user can configure it to whatever port they would like.
For the sake of similarity with other setup methods that all use port 8000, I would like to keep this the same.
In addition, the port the agent uses need to be updated as well.